auslib/blobs/schemas/apprelease-v8.yml

```diff
@@ -0,0 +1,226 @@
+                            properties:
+                              filesize:
+                                oneOf:
+                                  -
+                                    type: number
+                                    description: The filesize of this partial update
+                                    pattern: ^[0-9]*$
+                                  -
+                                    type: string
+                                    description: The filesize of this partial update
+                                    pattern: ^[0-9]*$
+                              proof:
+                                type: string
+                                description: The inclusion proof required for binary transparency test.
+                              from:
+                                type: string
+                                description: The release this partial update can be applied to. Should match another release in Balrog"s database.
+                              hashValue:
+                                type: string
+                                description: The hash value of this partial update, calculated using the hashFunction specified at the top level.
+                              fileUrl:
+                                type: string
+                                description: The URL to this partial update. If specified, overrides anything set in the top level fileUrls.
+                                format: uri
+                        completes:
+                          type: array
+                          items:
+                            type: object
+                            required:
+                              - filesize
+                              - from
+                              - hashValue
+                              - proof
+                            additionalProperties: false
+                            properties:
+                              filesize:
+                                oneOf:
+                                  -
+                                    type: number
+                                    description: The filesize of this complete update
+                                    pattern: ^[0-9]*$
+                                  -
+                                    type: string
+                                    description: The filesize of this complete update
+                                    pattern: ^[0-9]*$
+                              proof:
+                                type: string
+                                description: The inclusion proof required for binary transparency test.
```

Comment:
Are there any length requirements around these? If so, it would be good to enforce them here.